packages/python-yarl/python-yarl.spec

```diff
@@ -6,18 +6,15 @@
 
 Name:           %{?scl_prefix}python-%{pypi_name}
 Version:        1.7.2
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        Yet another URL library
 
 License:        Apache 2
 URL:            https://github.com/aio-libs/yarl/
 Source0:        https://files.pythonhosted.org/packages/source/y/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
 
 BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-devel
-BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-idna >= 2.0
-BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-multidict >= 4.0
 BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-setuptools
-BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-typing-extensions >= 3.7.4
```

Comment:
I'm curious as to why we are dropping dependencies on what sounds like a simple rebuild

Reply:
good enough for me.